internal/service/config/nginx_config_service.go

```diff
@@ -145,13 +151,24 @@ func (n *Nginx) ParseConfig(ctx context.Context) (any, error) {
 
 	stubStatus, err := n.stubStatus(ctx, payload)
 	if err != nil {
-		slog.Warn("Can not set stub_status", "error", err)
+		slog.WarnContext(ctx, "Unable to get Stub Status API from configuration, NGINX OSS metrics will be "+
```

Comment:
Can we change the log statement here to just `Unable to get Stub Status API from NGINX configuration`?
The reason being is we haven't decided yet on how we are going to collect NGINX metrics.
Also can this be a debug message instead of a warning?